examples/fitness/lib/settings.dart

```diff
@@ -29,7 +28,7 @@ class SettingsFragmentState extends State<SettingsFragment> {
     return new ToolBar(
       left: new IconButton(
         icon: "navigation/arrow_back",
-        onPressed: config.navigator.pop
+        onPressed: Navigator.of(context).pop
```

Comment:
I'd break this out into an anonymous closure so we don't do the walk every build.